components/shared_code/src/shared/utils/env.py

```diff
@@ -0,0 +1,19 @@
+"""Environment functions."""
+
+import os
+
+from dotenv import load_dotenv
+
+
+def getenv(key: str) -> str:
+    """Return the environment variable's value as string."""
+    return os.getenv(key, "")
+
+
+def loadenv(*filenames: str) -> None:
+    """Load the environment variables from the files.
+
+    By default, load_dotenv doesn't override existing environment variables so pass file with the defaults last.
```

Comment:
So a user would need to specify all settings, since the defaults contain example values?

Reply:
No, the defaults.env files contain the default values for the environment variables, as suggested by the name ;-)